Library/Formula/kafkacat.rb

```diff
@@ -11,11 +11,22 @@ class Kafkacat < Formula
     sha256 "a66d6d312498e117d485d7eea58e412da287be1dc08a1be0cd621d6a77763c7f" => :mavericks
   end
 
+  option "with-json", "Adds JSON support to kafkacat"
+
   depends_on "librdkafka"
+  depends_on "yajl" if build.with?("json")
```

Comment:
`depends_on "yajl" => :optional` and a `option "with-yajl", "Adds JSON support"` are preferable because of our crappy dependency system.